homeassistant/components/roborock/sensor.py

```diff
@@ -76,6 +76,25 @@ class RoborockSensorDescription(
     ),
 ]
 
+CLEAN_INFORMATION_SENSORS = [
+    RoborockSensorDescription(
+        native_unit_of_measurement=UnitOfTime.SECONDS,
+        key="cleaning_time",
+        translation_key="cleaning_time",
+        icon="mdi:progress-clock",
```

Comment:
This already has a device class, we should not have to set an icon in this case.

Reply:
Thank you. @Lash-L can you address in a followup PR?

Reply:
Will do once I am back at a computer. Thanks for pointing this out

Reply:
Fixed with #93774